ledger/simulation/testing/utils.go

```diff
@@ -76,8 +76,24 @@ func (info TxnInfo) InnerTxn(parent transactions.SignedTxn, inner txntest.Txn) t
 	return inner
 }
 
-// PrepareSimulatorTest creates an environment to test transaction simulations
-func PrepareSimulatorTest(t *testing.T) (l *data.Ledger, accounts []Account, txnInfo TxnInfo) {
+// Environment contains the ledger and testing environment for transaction simulations
+type Environment struct {
```

Comment:
you don't want to go with SimulateEnvironment or SimulationEnvironment or SimEnvironment

Reply:
Other packages will access this as `simulationtesting.Environment`, so I think that's qualified enough